jenkins_pipelines/manager_prs/Jenkinsfile_prs_backend_unitpsql

```diff
@@ -58,9 +65,11 @@ pipeline {
         }
     }
   post { 
-        success {
```

Comment:
Isn't this going to run even when the test fails?

It's not clear to me by having a look at https://jenkins.io/doc/book/pipeline/syntax/#post

IMHO a better approach (and easier to understand) would be:

```
post { 
      success {
            if (params.cleanWorkspace == true) {
                echo 'Clean up current workspace, when job succeeds.'
                cleanWs()
            }
      }
}
```

Reply:
You are right, it should be only when success, that's a mistake.
Anyhow the correct way is:

````
  post { 
        success{
            script {
                if (params.cleanWorkspace == true) {
                    echo 'Clean up current workspace, when job success.'
                    cleanWs()
                }
            }
        }
  }
```

Reply:
No problem :-)

I didn't made corrections there because I suddenly saw the new PR :-\